src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -559,7 +561,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 			throw new Error('Terminal disposed of during xterm.js creation');
 		}
 
-		const xterm = this._instantiationService.createInstance(XtermTerminal, Terminal, this._configHelper, this._cols, this._rows);
+		const xterm = this._instantiationService.createInstance(XtermTerminal, Terminal, this._configHelper, this._cols, this._rows, this.target || TerminalLocation.Panel);
```

Comment:
if target is undefined for the instance, should we throw or fallback to panel?

Reply:
Fallback seems fine